deps/jemalloc/include/jemalloc/internal/jemalloc_internal_inlines_c.h

```diff
@@ -228,18 +232,20 @@ iget_defrag_hint(tsdn_t *tsdn, void* ptr) {
 		extent_t *slab = iealloc(tsdn, ptr);
 		arena_t *arena = extent_arena_get(slab);
 		szind_t binind = extent_szind_get(slab);
-		bin_t *bin = &arena->bins[binind];
-		malloc_mutex_lock(tsdn, &bin->lock);
+		unsigned shardind = extent_binshard_get(slab);
```

Comment:
This function (iget_defrag_hint) is the only change made not a part of jemalloc 5.2.1